Python/Batch/common/helpers.py

```diff
@@ -516,3 +516,49 @@ def wrap_commands_in_shell(ostype, commands):
         return 'cmd.exe /c "{}"'.format('&'.join(commands))
     else:
         raise ValueError('unknown ostype: {}'.format(ostype))
+
+
+def wait_for_job_creation(batch_client, job_schedule_id, timeout):
```

Comment:
function is called `wait_for_job_creation`, when it is really a job_schedule that is being waited for